crates/extensions/c8y_config_manager/src/actor.rs

```diff
@@ -64,7 +65,7 @@ impl ConfigManagerActor {
         &mut self,
         message: MqttMessage,
         message_box: &mut ConfigManagerMessageBox,
-    ) -> Result<(), anyhow::Error> {
+    ) -> Result<(), DynError> {
```

Comment:
That's my bad. I'll do a follow-up PR to fix this by makes all config manager functions return `ConfigManagementError` instead of this `DynError`.

Reply:
It would even be good to do this in that PR.

Reply:
Done as a fixup commit https://github.com/thin-edge/thin-edge.io/pull/1752/commits/f9fd119c2ce26bbf19756a4c8f88e0b418ad1a01